python/orca/tutorial/NCF/process_xshards.py

```diff
@@ -143,7 +143,7 @@ def rename(df, col):
     train_data, test_data = data.transform_shard(split_dataset).split()
 
     feature_cols = get_feature_cols()
-    label_cols = ["label"]
+    label_cols = get_label_cols()
```

Comment:
remove these two lines and directly call get_feature_cols() and get_label_cols() in return?